packages/astro/package.json

```diff
@@ -163,6 +163,7 @@
     "http-cache-semantics": "^4.1.1",
     "js-yaml": "^4.1.0",
     "kleur": "^4.1.5",
+    "linkedom": "^0.18.4",
```

Comment:
Is this a typo?

Reply:
Yes! Lowercase `-d`
![image](https://github.com/user-attachments/assets/570d23e4-dd2b-418e-8d3f-ea17146c7d3a)